rust_src/src/process.rs

```diff
@@ -537,4 +538,24 @@ pub fn list_system_processes_rust() -> LispObject {
     unsafe { list_system_processes() }
 }
 
+/// Interrupt process PROCESS
+/// PROCESS may be a process, a buffer, or the name of a process or buffer.
+/// No arg or nil means current buffer's process.
+/// Second arg CURRENT-GROUP non-nil means send signal to
+/// the current process-group of the process's controlling terminal
+/// rather than to the process's own process group.
+/// If the process is a shell, this means interrupt current subjob
+/// rather than the shell.
+
+/// If CURRENT-GROUP is `lambda', and if the shell owns the terminal,
+/// don't send the signal.
+
+/// This function calls the functions of `interrupt-process-functions' in
+/// the order of the list, until one of them returns non-`nil'.  */)
+#[lisp_fn(name="interrupt-process")]
```

Comment:
```suggestion
#[lisp_fn(min = "0")]
```

After `Sinterrupt_process` there are 3 numbers. They represent the minimum number of required arguments, the maximum allowed, and finally whether the DEFUN is interactive. Zero here works like a NULL. Otherwise there will be a string.